presidio-analyzer/analyzer/nlp_engine/spacy_nlp_engine.py

```diff
@@ -18,7 +18,7 @@ def __init__(self):
         self.logger = logging.getLogger(__name__)
         self.logger.setLevel(loglevel)
 
-        self.logger.info("Loading NLP model...")
+        logging.info("Loading NLP model...", extra={'req_id': ''})
```

Comment:
can req_id have a default value? We could keep all non-request-related logs as is

Reply:
a) no it is not possible unfortunately.
b) I chose a different path so (a) is no longer an issue